InjectionBundle/SwiftInjection.swift

```diff
@@ -118,6 +118,10 @@ public class SwiftInjection: NSObject {
             if (classMetadata.pointee.Data & 0x1) == 1 {
                 if classMetadata.pointee.ClassSize != existingClass.pointee.ClassSize {
                     NSLog("💉 \(oldClass) metadata size changed. Did you add a method?")
+
+                    if !(newClass.superclass() == NSObject.self) {
+                        NSLog("💉 ⚠️ Adding or removing functions on pure Swift classes is not supported. Your application will eventually crash. Inherit from NSObject while developing to avoid unexpected crashes. ⚠️")
+                    }
                 }
```

Comment:
No point in recommending inheriting from NSObject as it will never be reliable.

I’d suggest "Adding or removing methods on Swift classes is not supported. Your application will likely crash."

Reply:
Interesting, I've had success with adding and removing methods when the objects inherit from NSObject.
I might just have been lucky with my injections or?

We can totally change the message if you like, I just thought that new methods got swizzled in when they are available in Objective-C runtime.

Reply:
can you change it to a print so people don’t miss the message?

Reply:
Can you take out the newClass.superclass() == NSObject.self and change the previous message?

Reply:
But then it shouldn't say Swift classes as it applies to everything?

Reply:
If it goes down that path it is a swift class. This is what the "classMetadata.pointee.Data & 0x1) == 1" does. Perhaps that deserves a comment :)

Reply:
That’s how they added Swift into the Objective-C runtime with backward compatibility.

Reply:
Getting there. Lets fix up the comments. How about moving the comment "// Swift equivalent of Swizzling” to the line after the if and add a new comment the line before the if “// Is this a Swift class?”.

Reply:
Push a new commit now, was that what you meant?

Reply:
Exactly.. we’re good to go.